src/IssueViz.UnitTests/Editor/Common/CommonTaggerProviderTestsBase.cs

```diff
@@ -46,7 +46,7 @@ public void CreateTagger_BufferIsNull_Throws()
         }
 
         [TestMethod]
-        public void CreateTagger_BufferIsNotNull_ReturnsSingletonTagger()
+        public void CreateTagger_BufferIsNotNull_ReturnsUniqueTaggerPerRequest()
```

Comment:
Maybe `new` instead of `unique`? It will be more evident that it's a new instance each time.